community/dnscrypt-proxy/config-full-paths.patch

```diff
@@ -265,7 +305,7 @@ index 0000000..347ada5
 +
 +  ## Path to the query log file (absolute, or relative to the same directory as the executable file)
 +
-+  # file = '/var/log/query.log'
++  # file = '/var/log/dnscrypt-proxy/query.log'
```

Comment:
If you use a directory here, be sure that is created in  the APKBUILD during the installation, and with the correct permissions.

Reply:
I'd left that to the init.d script which has the following to create if doesn't exist :
checkpath -q -d -m 0775 -o "${command_user}" \
		/var/cache/"${RC_SVCNAME}" \
		/var/log/"${RC_SVCNAME}"

Is that OK, or would you prefer a mkdir in the pre-install as well?

Reply:
It's better to create it in the APKBUILD itself, since you might run the dnscrypt without using the init.d